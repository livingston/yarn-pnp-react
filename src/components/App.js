import React from 'react';
import { hot } from 'react-hot-loader/root'; // eslint-disable-line import/no-extraneous-dependencies
import {
  BrowserRouter as Router,
  Route, Switch, Link
} from 'react-router-dom';

import List from './List';

function App() {
  return (<Router>
    <nav>
      <ul>
        <li><Link to="/">Home</Link></li>
        <li><Link to="/list">Users</Link></li>
      </ul>
    </nav>
    <section>
      <Switch>
        <Route path="/list" component={List} />
      </Switch>
    </section>
  </Router>);
}

export default hot(App);
