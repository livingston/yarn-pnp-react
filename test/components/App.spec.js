import React from 'react';
import { render } from '@testing-library/react';

import App from 'components/App';

describe('App', () => {
  it('should render the component', () => {
    const { container } = render(<App />);

    expect(container).toMatchSnapshot();
  });
});
