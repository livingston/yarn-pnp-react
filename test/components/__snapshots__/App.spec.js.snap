// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the component 1`] = `
<div>
  <nav>
    <ul>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/list"
        >
          Users
        </a>
      </li>
    </ul>
  </nav>
  <section />
</div>
`;
